client/web/src/enterprise/batches/create/workspaces-preview/WorkspacesPreview.tsx

```diff
@@ -27,6 +27,19 @@ const ON_STATEMENT = `on:
   - repositoriesMatchingQuery: repo:my-org/.*
 `
 
+const WAITING_MESSAGES = [
+    'Hang tight while we look for matching workspaces...',
+    'Still searching, this should just take a moment or two more...',
+    '*elevator music* (Still looking for matching workspaces...)',
+    "This is getting embarrassing, but we're still searching...",
```

Comment:
```suggestion
    "This is getting embarrassing, but we're still searching...",
```

Can we remove this one or find some other wording? 😆 It makes it sound like we are at blame, where it could really be that they have an expensive query going on there.

Reply:
Hahaha fair, we'd *never* be the ones to blame after all. 😜